src/components/scenes/FioAddressDetailsScene.js

```diff
@@ -84,12 +84,12 @@ class FioAddressDetails extends React.Component<Props, LocalState> {
     } = this.props
     const styles = getStyles(theme)
 
-    if (bundlesIsAlmostSpent(bundles)) {
+    if (bundles < BUNDLES_AMOUNT_ALERT) {
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name